third_party/closure-compiler/runner/src/AmpCommandLineRunner.java

```diff
@@ -0,0 +1,34 @@
+import com.google.common.collect.ImmutableSet;
+import com.google.javascript.jscomp.CommandLineRunner;
+import com.google.javascript.jscomp.CompilerOptions;
+import com.google.javascript.jscomp.CustomPassExecutionTime;
+
+public class AmpCommandLineRunner extends CommandLineRunner {
+  
+  ImmutableSet<String> suffixTypes = ImmutableSet.of(
+      "log.dev.fine",
+      "log.dev.assert"
+      );
+
+  protected AmpCommandLineRunner(String[] args) {
+    super(args);
+  }
+
+  @Override
+  protected CompilerOptions createOptions() {
+    CompilerOptions options = super.createOptions();
+    AmpPass ampPass = new AmpPass(getCompiler(), suffixTypes);
+    options.addCustomPass(CustomPassExecutionTime.BEFORE_OPTIMIZATIONS, ampPass);
```

Comment:
@erwinmombay @cramforce Not sure if there's a more direct facility in closure compiler, but generally this rings true to do it `BEFORE_OPTIMIZATIONS` since stripping calls can lead to more things to optimize...